assignment/scripts/function-practice.js

```diff
@@ -51,32 +52,55 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 //    array is empty, return `undefined`.
 function getLast( array ) {
 
+if (array.length === []) {
```

Comment:
Not quite. First, remember that `array.length` is a number. So it is not going to be equal to the object `[]`. Second, since `[]` is an object, it will _never_ equal anything other than itself.

So this if block will never run, but thankfully it turns out you don't need it. Line 58 will return `undefined` all on its own if there are no elements to get out of the array.